perception/autoware_map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -397,6 +392,20 @@ bool withinRoadLanelet(
   return false;
 }
 
+bool withinRoadLanelet(
+  const TrackedObject & object, const lanelet::LaneletMapPtr & lanelet_map_ptr,
+  const bool use_yaw_information = false)
+{
+  const auto & obj_pos = object.kinematics.pose_with_covariance.pose.position;
+  lanelet::BasicPoint2d search_point(obj_pos.x, obj_pos.y);
+  // nearest lanelet
+  constexpr double search_radius = 10.0;  // [m]
+  const auto surrounding_lanelets_with_dist =
+    lanelet::geometry::findNearest(lanelet_map_ptr->laneletLayer, search_point, search_radius);
```

Comment:
This is also another misusage, so please replace with
```suggestion
    lanelet::geometry::findWithin2D(lanelet_map_ptr->laneletLayer, search_point, search_radius);
```